.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,44 @@
-**Description:**
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
 
+## Description
+<!--- Describe your changes. -->
 
-**Related issue (if applicable):** fixes #
+## Types of Changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
These boxes actually get listed as tasks, so by clicking 1/3 of them it will always indicate your status is not 100% complete